packages/sdk/src/lib/dataProtectorSharing/consumeProtectedData.ts

```diff
@@ -151,11 +160,11 @@ export const consumeProtectedData = async ({
     let tx;
     let transactionReceipt;
     try {
-      // TODO: when non free workerpoolorders is supported add approveAndCall (see implementation of buyProtectedData/rentProtectedData/subscribeToCollection)
```

Comment:
keep the commentary or you should implement the ApproveAndCall for the consomeProtectData